firmware/Platypus.cpp

```diff
@@ -215,6 +241,7 @@ void Motor::disable()
 
 float Motor::current()
 {
+  //Will no longer work with battery directly powering esc?
```

Comment:
This will no longer reflect the motor current, but it now reflects the pump or airgateway current, so it's probably fine to leave it as-is.